test/parallel/test-internal-util-decorate-error-stack.js

```diff
@@ -32,7 +33,7 @@ function checkStack(stack) {
 }
 let err;
 const badSyntaxPath =
-    path.join(common.fixturesDir, 'syntax', 'bad_syntax')
+    path.join(fixtures.fixturesDir, 'syntax', 'bad_syntax')
```

Comment:
I think this should actually use the `fixtures.path` function.

Reply:
For instance,

```js
const badSyntaxPath = fixtures.path('syntax', 'bad_syntax');
```